src/core/ddsc/include/dds/ddsc/dds_public_status.h

```diff
@@ -422,7 +422,7 @@ dds_get_liveliness_changed_status (
  */
 DDS_EXPORT dds_return_t
 dds_get_sample_rejected_status (
-  dds_entity_t reader,
```

Comment:
This really only takes a reader, so I think "reader" is the better choice.

Reply:
Applied and fixed.